students/kpersic/module_3/blog/scss/_icons.scss

```diff
@@ -0,0 +1,11 @@
+.fa-pencil {
+    margin-right: 9.6px;
+}
+
+.fa-arrow-right {
+    margin-left: 8px;
```

Comment:
Kada smo već kod margina, preporučio bih ti da napraviš nekakvu spacing skalu za margine koju možeš koristiti poput varijabli. To je super korisno pogotovo kada imaš puno vrijednosti koje se ponavljaju (kao što imaš sada) jer kasnije ako dođe do potrebe da malo podesiš veličine margina, možeš napraviti promjenu na jednom mjestu i sve će se proporcionalno smanjiti.

Na primjer, napraviš varijablu `$margin-1: 8px-;` i onda ju koristiš ovdje i drugdje.

Padding često zna imati vlastiti skup veličina jer ne raste toliko koliko margine. Vrijednosti paddinga znaju biti puno manjeg raspona npr. 4-48px dok margine znaju biti velike od 4-256px ako ne i veće nekada.